### PR TITLE
[LC-461] Manage dependencies

### PR DESCRIPTION
**Pull request by metalg0su**
Please check package metadata by `$ python setup.py egg_info` (loopchain.egg_info/PKG-INFO)

### CODE REVIEW

**Comment by metalg0su** [comment]
Need precious reviews.

File: Makefile [hunk 1]
```diff
@@ -45,16 +49,20 @@ requires:
...
 
-install-dev: requires
-	$(PIP_INSTALL_REQUIREMENTS)[tests]
-
 # Generate python gRPC proto and generate a key
 generate: generate-proto generate-key
 
 # Generate python gRPC proto
 generate-proto:
+	@python3 -m venv ${PROTO_VENV}
```
Comment (metalg0su) [inline Makefile]: Buggy when executed this line inside `virtualenv`

File: setup.py [hunk 2]
```diff
@@ -41,7 +93,14 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3 :: Only'
-    ]
+    ],
+    'keywords': '',  # FIXME or delete
```
Comment (metalg0su) [inline setup.py]: https://packaging.python.org/guides/distributing-packages-using-setuptools/#keywords

File: setup.py [hunk 3]
```diff
@@ -12,23 +22,65 @@
     with open(os.path.join('.', 'VERSION')) as version_file:
         version = version_file.read().strip()
 
+# Source Branch for Development
+dev_branch_iconservice = "master"
+dev_branch_iconcommons = "master"
+dev_branch_iconrpcserver = "develop"
+dev_branch_tbears = "master"
```
Comment (metalg0su) [inline setup.py]: .

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+[options.extras_require]
+icon =
+    iconservice
+    iconcommons
+    iconrpcserver
+    tbears
+icon_dev =
+    iconservice @ git+https://github.com/icon-project/icon-service.git@develop
+	iconcommons @ git+https://github.com/icon-project/icon-commons.git@master
+	iconrpcserver @ git+https://github.com/icon-project/icon-rpc-server.git@develop
```
Comment (yakkle) [inline setup.cfg]: Please use unified indent type(`tab` or `space`?)
I recommend `space` if it has no problem.

Comment (metalg0su) [reply setup.cfg]: Thank you.

**Comment by metalg0su** [comment]
Need to review:
- Is it ok that we manage dependencies at setup.cfg?

**Pull request closed without merging**